language/move-prover/doc/user/spec-lang.md

```diff
@@ -206,6 +212,67 @@ The choice also comes in a form to select the *minimal* value from a set of inte
 choose min i: num where in_range(v, i) && v[i] == 2
 ```
 
+## Cast Operator
+
+In the specification language, we can use the same syntax `(e as T)` to cast an expression `e` with one integer type to
+`T`, an integer type of another size.
+
+
+## Shift Operator
+
+Shift operators `<<` and `>>` are supported in the specification language and both of them have the same semantics (including abort) with the Move language.
```

Comment:
Could you add a bit explanation about the aborting condition of the shift operators?